src/proxy.ts

```diff
@@ -1,12 +1,53 @@
 import express from 'express';
-import { createProxyServer } from 'http-proxy';
+import { ClientRequest } from 'http';
```

Comment:
Good to know, fixed some existing import ;)

Reply:
I was talking only about modules that are part of Node.js (not Express imports) and this ESLint rule: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/import-style.md

Example:

![image](https://user-images.githubusercontent.com/643434/142174677-b7bb0808-34ac-4d37-9fa7-53598c70aed3.png)

==>

![image](https://user-images.githubusercontent.com/643434/142174704-a9b6b95d-3f84-4427-b055-b51431269c7b.png)


What I've missed is that this rule applies only for `path`, `chalk` and `util`: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v38.0.1/rules/import-style.js#L123-L131
Not for `http` (which is also part of Node).

The "future" eslint-plugin-unicorn rule I'm talking about is this one: https://github.com/pmu-tech/stub-server/blob/v0.5.0/.eslintrc.js#L88

![image](https://user-images.githubusercontent.com/643434/142175456-fc92e45a-161b-466c-8c0e-deef84faf950.png)

`node:` protocol (aka `node:` imports): https://github.com/nodejs/node/issues/38343#issue-864698633

Reply:
For the Express changes, I wanted to make them similar as the other Express imports because all imports on the project were `import express from 'express';`.

For the node protocol, when do we plan to update the imports and turn the rule "on" anytime soon ?